hardhat.config.js

```diff
@@ -62,21 +62,21 @@ module.exports = {
         disambiguatePaths: false
     },
     abiExporter: {
-        path: './solidity/build/abi',
+        path: './build/abi',
         clear: true
     },
 
     // System Config
     paths: {
-        sources: './solidity/contracts',
-        tests: './solidity/test',
+        sources: './contracts',
```

Comment:
IMO, this is a good opportunity to rename folder `test` (Truffle's default) to `tests`.